MCLauncher/VersionList.cs

```diff
@@ -27,6 +27,14 @@ class VersionList : ObservableCollection<WPFDataTypes.Version> {
             foreach (JArray o in data.AsEnumerable().Reverse()) {
                 Add(new WPFDataTypes.Version(o[1].Value<string>(), o[0].Value<string>(), o[2].Value<int>() == 1, _commands));
             }
+            string[] subdirectoryEntries = Directory.GetDirectories(".");
+            foreach (string subdirectory in subdirectoryEntries)
+            {
+                if (subdirectory.Contains("(imported)"))
```

Comment:
This should probably be replaced with a scan for directories matching the following criteria: has AppxManifest.xml and Minecraft.Windows.exe (or the older one, how was it called?) inside.